cairo/tests/ethereum/cancun/vm/test_interpreter.py

```diff
@@ -14,18 +14,33 @@
 from tests.utils.message_builder import MessageBuilder
 from tests.utils.strategies import environment_lite
 
+# TODO: enable execution of these precompiles
+unimplemented_precompiles = [
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+]
+
 message_without_precompile = (
     MessageBuilder()
     .with_current_target(
-        st.integers(min_value=11, max_value=2**160 - 1)
+        st.integers(min_value=0, max_value=2**160 - 1)
+        .filter(lambda x: x not in unimplemented_precompiles)
```

Comment:
not sure about that.
Once unimplemented_precompiles is empty this strategy will have precompiles?

Reply:
yes